### ISSUE TITLE
[website] Color contrast on gray text is insufficient

### ISSUE THREAD

**Issue by asadotzler** [mozilla/fx-private-relay#222]
Steps to reproduce: Visit https://relay.firefox.com/accounts/profile/ and run the contrast audit from the Firefox DevTools. 

Results: All of the gray text fails contrast requirement (ratio of 3.99:1)

Expected: All of the gray text passes (ratio of 4.5:1 or better)

This gray text seems to all be #7b7f8d on a white background. To meet requirements, the gray text should be something like #727583 or darker.
